onnxruntime/contrib_ops/cpu/maxpool_with_mask.h

```diff
@@ -35,8 +35,9 @@ class MaxpoolWithMask : public OpKernel, public PoolBase {
 
     std::vector<int64_t> pads = pads_;
     std::vector<int64_t> kernel_shape = kernel_shape_;
+    std::vector<int64_t> dilations = dilations_;
```

Comment:
is this copy required?

Reply:
:smile: 

---
In reply to: [277976697](https://github.com/Microsoft/onnxruntime/pull/873#discussion_r277976697) [](ancestors = 277976697)

Reply:
Removed copy

Reply:
did you forget to remove this copy?

Reply:
looks like this was missed because it was contrib ops folder (previously fixed files under providers folder). Fixed now.